azure-pipelines-templates/device-test.yml

```
@@ -0,0 +1,154 @@
+
+    pool:
+      name: TestStream
+      demands: 
+        - ${{ parameters.appComponents }}
+
+    jobs:
+      - job:
+        displayName: 'Test job for ${{ parameters.appComponents }}'
+
+        steps:
+        - task: DownloadPipelineArtifact@2
```

yamllint

[warning] wrong indentation: expected 10 but found 8

(indentation)